Packs/CommonScripts/Scripts/ParseEmailFilesV2/ParseEmailFilesV2.yml

```diff
@@ -0,0 +1,128 @@
+args:
+- default: true
+  description: Entry ID with the Email as a file in msg or eml format
+  name: entryid
+  required: true
+- auto: PREDEFINED
+  defaultValue: "false"
+  description: Will parse only the headers and return headers table
+  name: parse_only_headers
+  predefined:
+  - "true"
+  - "false"
+- defaultValue: "3"
+  description: How many levels deep we should parse the attached emails (e.g. email
+    contains an emails contains an email). Default depth level is 3. Minimum level
+    is 1, if set to 1 the script will parse only the first level email
+  name: max_depth
+- required: false
+  name: forced_encoding
+  description: Use only the force encoding when parsing the message, e.g 'iso-2022-jp'.
+    Relevant to msg files only.
+- required: false
+  name: default_encoding
+  description: Use only the default encoding when parsing the message with the detected
+    encoding fails, e.g 'utf-8'. Relevant to msg files only.
```

Comment:
Also this does not appear in code

Reply:
good catch
I forgot to handle it in the code